packages/icons/general/Loading.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 50 56">
```

Comment:
ditto.

But we already have some other loading icons

<img width="499" alt="image" src="https://github.com/DimensionDev/Maskbook/assets/1141198/2da1fe92-a69d-4047-b1b6-19e6d0556bad">